src/routes/blog/post/building-init-giveaway-app/+page.markdoc

```diff
@@ -384,4 +383,4 @@ The giveaway roulette app was used over 450 times by 230+ different users across
 
 You can find the application’s complete source code at this [GitHub Repo](https://github.com/adityaoberai/InitGiveawayRoulette).
 
-[Join us on Discord](https://appwrite.io/discord) to be the first to get updates and to be part of a vibrant community!
+[Join us on Discord](https://appwrite.io/discord)&nbsp;to be the first to get updates and to be part of a vibrant community!
```

Comment:
I saw some html entities added to md files, was it the formatter or did you add the non breaking spaces yourself?

Reply:
I didn't add these, I think it's from formatting.